server/importers/index.js

```diff
@@ -68,8 +91,9 @@ class Importer {
     } else {
       console.warn('DB already created - skipping download & unzip.')
     }
-    await this.shapes()
+    // await this.shapes()
```

Comment:
```suggestion
    await this.shapes()
```
Think you may have accidentally commented this one out.